charm/ksql/templates/log4j.properties

```diff
@@ -0,0 +1,16 @@
+log4j.rootLogger=INFO, stdout
```

Comment:
Does log4j output get sent to journald with this configuration?

Reply:
Will need to research this, added TODO